quipucords/scanner/network/inspect_callback.py

```diff
@@ -94,113 +44,87 @@ class InspectResultCallback(CallbackBase):
     """
 
     # pylint: disable=protected-access
-    def __init__(self, scan_task, display=None):
+    def __init__(self, scan_task, idx, group_total):
         """Create result callback."""
-        super().__init__(display=display)
         self.scan_task = scan_task
         self.source = scan_task.source
-        self.results = []
         self._ansible_facts = {}
+        self.last_role = None
+        self.idx = idx
+        self.group_total = group_total
 
-    # Ansible considers tasks failed when their return code is
-    # nonzero, even if we set ignore_errors=True. We want to be able
-    # to process those task results, so we need to handle failed
-    # results the same way we treat ok ones.
-    def v2_runner_on_ok(self, result):
-        """Print a json representation of the result."""
-        try:
-            self.handle_result(result)
-        except Exception as error:
-            self.scan_task.log_message(
-                'UNEXPECTED FAILURE in v2_runner_on_ok.'
-                '  Error: %s\nAnsible result: %s' % (
-                    error, result._result),
-                log_level=logging.ERROR)
-            traceback.print_exc()
-            raise error
-
-    def v2_runner_on_failed(self, result, ignore_errors=False):
-        """Print a json representation of the result."""
-        # pylint: disable=protected-access
-        try:
-            with_items = result._result.get('results') is not None
-            if not with_items and result._result.get('msg') is not None:
-                result_obj = _construct_result(result)
-                if result_obj.get('key') != 'internal_user_has_sudo_cmd':
```

Comment:
Instead of quieting the logging output through keys, I quieted the logs through the check `if event_dict.get('ignore_errors', False):`. That way it would actually only log unexpected ansible errors.